resources/howtos/digitalocean_droplet.md

```diff
@@ -0,0 +1,109 @@
+# How to deploy a MeiliSearch instance on DigitalOcean
+
+## Create an out-of-the-box MeiliSearch
+
+### 1. Create a new droplet
+
```

Comment:
Could you say in one sentence what a droplet is?

Reply:
Sure! Done :)